### PR TITLE
Cloudinary setup completed

### PR DESCRIPTION
**Pull request by wJoenn**

### CODE REVIEW

File: config/storage.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
+cloudinary:
+  service: Cloudinary
+  folder: <%= Rails.env %>
```
Comment (MarianaCarlota) [inline config/storage.yml:11]: Do you already have the claudinary url to put in the .env file?

Comment (wJoenn) [reply config/storage.yml:11]: The .env file is never pushed to github so you'll have to create it yourself. The key is the same as the previous project

**Review by AntonioDannemann — COMMENTED** [review]
what is the `has_one_attached :photo` how does it work?

**Comment by wJoenn** [comment]
This links the model to Cloudinary so that we can store the photo in Cloudinary and still be able to do Model.photo.key

**Comment by AntonioDannemann** [comment]
perfect, thanks

**Review by AntonioDannemann — APPROVED** [review]

**Pull request merged**